### ISSUE TITLE
Manifest error when getting image listing from a registry after docker 1.10 upgrade

### ISSUE THREAD

**Issue by rqc** [eedevops/shipyard#49]
Getting this error: 

```
400 Bad Request: {"errors":[{"code":"MANIFEST_INVALID","message":"manifest invalid","detail":{"description":"null"}}]}
```

We need to upgrade the code that parses the JSON payload from the registries to support the new manifest format.

**Comment by rqc** [comment]
Manifest documentation: https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md

**Comment by rqc** [comment]
Fixed in #62
